### DISCUSSION TITLE
main/qa/

### DISCUSSION THREAD

**Discussion by giscus** [THU-feiyue/THU-feiyue#48] [General]
# main/qa/

https://feiyue.online/main/qa/

**Comment by Shengsxiiao1025** [comment]
专家推荐信可以设立一个截止日期，如果专家无法在截止日期前完成推荐信，会怎么办啊？那时候把专家信息填写过早，忘了查看一些小提示了，请学长学姐们赐教，感激不尽。

**Reply by SamSeven777** [reply]
关于在网申系统中将推荐信的截止日期设置得过早，导致推荐人未能在截止日期前提交推荐信的问题：
1. 联系推荐人，说明情况，请求他们尽快提交推荐信，可能只作为提醒日期，提交链接并未过期。
2. 在网申系统中寻找修改推荐人信息和截止日期的选项，如无法修改可以尝试删除和重新添加推荐人信息，重新发送推荐信提交链接。
3. 如果已经无法自行修改推荐信信息，请立刻联系学校招生部门，询问解决办法，请求协助解决问题。

**Comment by Shengsxiiao1025** [comment]
感謝解惑！！！謝謝你們！感激不盡！

Sam7 ***@***.***> 于2023年11月29日周三 23:59写道：

**Comment by Shengsxiiao1025** [comment]
抱歉啊，再次詢問一個問題。

清華大學碩士生申請，需要HSK成績證明，但已知HSK成绩证明，若为中文授课本科学位者，可以同毕业学校出具的证明，获得所申请院系许可后，可免交HSK成绩证明，因此本人是否需事前将毕业学校出具的证明交予申请院系查证后，方才处理申请手续的栏目。

或是可于栏目处直接上传毕业学校出具的证明替代HSK成绩证明，若如此，关于HSK成绩总分及等级处该如何填写。

还望答疑解惑，词不达意之处，还请见谅，先此致谢！
